themes/default/layouts/blog/single.html

```diff
@@ -42,9 +42,6 @@ <h1 class="no-anchor"><a data-track="blog-title" href="{{ .Permalink }}">{{ .Tit
             <div class="md:w-3/12 md:pl-8">
                 <div class="sticky-sidebar">
                     {{ partial "blog/right-nav.html" . }}
-                    <div class="mt-8">
-                        <a href="{{ relref . "/pulumi-up" }}?iaid=blog-right-nav"><img src="/images/pulumiup/pulumiup-rectangle-cta.png" alt="Register now for PulumiUP" /></a>
-                    </div>
```

Comment:
Not necessarily for this PR, but since you've done the work of figuring out where to put these, replacing them with an empty shortcode could make it easier to drop things like this into the right places easily when/if we need to do this again someday.